try:
    pass


except ValueError as err:
    pass
except NameError as err:
    pass
except FileNotFoundError as err:
    pass
except:
    pass
